ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -445,14 +445,16 @@ function osd_activate {
   fi
   mkdir -p /var/lib/ceph/osd
   chown ceph. /var/lib/ceph/osd
+  # resolve /dev/disk/by-* names
+  ACTUAL_OSD_DEVICE=$(readlink -f ${OSD_DEVICE})
```

Comment:
this assumes OSD_DEVICE is a link, can you instead do something like `if [[ "$(find <path to device> -type l) -gt 0 ]]; then`, this will determine if the file is a link or not.